.dockerignore

```diff
@@ -0,0 +1,2 @@
+**/*.*
```

Comment:
What does this file do?

Reply:
By default the whole folder where the "docker build" is executed is send to the docker daemon as "context". This is not needed here  (as we do not COPY in the Dockerfile)  and its slow. 
Not sure, what VsCode does with it.

Reply:
Can we move this file into the devcontainer folder or is this right here?

Reply:
It depends on which "folder" VsCode executes the "docker build " . I suppose on the "root".
It needs to be the root of the "build context". I have a look